manager/integration/tests/common.py

```diff
@@ -1350,6 +1350,52 @@ def delete_replica_processes(client, api, volname):
             assert deleted
 
 
+def crash_replica_processes(client, api, volname):
+    replica_managers = []
+
+    volume = client.by_id_volume(volname)
+    for r in volume["replicas"]:
+        replica_managers.append(r["instanceManagerName"])
+
+    kill_command = [
+        '/bin/sh', '-c',
+        "kill `ps aux | " +
+        "grep 'longhorn replica' | grep -v grep | awk '{print $2}'`"
```

Comment:
This would kill all the replicas in the instance manager?

Reply:
In this case, there should be only one `attached` volume and each replica manager contains only one replica process

Reply:
This will break the stress test since we're running the test for multiple volumes at once.